test/units/TEST-04-JOURNAL.stopped-socket-activation.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+systemctl stop systemd-journald.service
+systemd-cat date && sleep 1
```

Comment:
sleep 1 seems a bit fragile here, maybe something like:

`timeout 30 bash -xec 'until test "$(systemctl show -p SubState --value systemd-journald.service)" = "running"; do sleep 1; done'`

Reply:
Thanks for the suggestion.